x-pack/platform/plugins/shared/fleet/server/services/agents/agentless_agent.ts

```diff
@@ -712,6 +827,10 @@ class AgentlessAgentService {
           message:
             'You have reached the limit for agentless provisioning. Please remove some or switch to agent-based integration.',
         },
+        list: {
+          log: '[Agentless API] Listing agentless deployments failed with a status 429, agentless agent limit reached.',
+          message: `The Agentless API could not list the agentless deployments because the request timed out. Please try again or contact your administrator.`,
```

Comment:
Is it possible to get a 429 for the agent limit on a listing request? Would we get a 429 for rate limiting?